src/bridge/IrcHandler.ts

```diff
@@ -148,19 +148,39 @@ export class IrcHandler {
      * @return {Promise} which is resolved when the PM room has been created.
      */
     private async createPmRoom (toUserId: string, fromUserId: string, fromUserNick: string, server: IrcServer) {
+        const users: {[userId: string]: number} = { };
+        users[toUserId] = 10;
+        users[fromUserId] = 100;
         const response = await this.ircBridge.getAppServiceBridge().getIntent(
             fromUserId
         ).createRoom({
             createAsClient: true,
             options: {
                 name: (fromUserNick + " (PM on " + server.domain + ")"),
                 visibility: "private",
-                preset: "trusted_private_chat",
+                // We deliberately set our own power levels below.
+                // preset: "trusted_private_chat",
```

Comment:
I don't understand this line.
Set our own power levels below *what*? The other user?
Why is the next line commented then? Do we need to keep this?

Reply:
Ah, perhaps this needs some clarity. We used to use the [preset `trusted_private_chat`](https://matrix.org/docs/spec/client_server/r0.6.0#post-matrix-client-r0-createroom) for creating a PM room, which basically means both users get powerlevel 100.

Because we want to limit the matrix user's ability to invite into a PM, we deliberately make them have a lower PL and increase the required PL to invite. This is done in the state object below. I've kept this comment and the commented out preset line to indicate that this is what we would be doing, if not for the special requirements.

I tend to prefer leaving this in so as to signal that there was a deliberate reason not to use the preset, but it's a taste thing and the comment can be removed/amended if needed.